doc/build/dts/howtos.rst

```diff
@@ -122,21 +122,18 @@ that the device is ready to be used before passing it to any API functions.
 
 In some situations the device cannot be known at build-time, e.g., if it depends
 on user input like in a shell application. In this case you can get the
-``struct device`` by combining :c:func:`DT_LABEL` with
+``struct device`` by combining :c:func:`DEVICE_DT_NAME` with
 :c:func:`device_get_binding`:
```

Comment:
`DEVICE_DT_NAME` doesn't really help if we depend on user input, so this paragraph isn't really helpful as it is written now.

Reply:
Updates